doc/release-notes-1.10.rst

```diff
@@ -67,7 +67,11 @@ Networking
 Bluetooth
 *********
 
-* details ...
+* Multiple qualification-related fixes for Bluetooth Mesh
+* Support for Bluetooth Mesh Friend Node role
+* Support for Bluetooth Mesh Foundation Client Models
+* New Bluetooth Mesh shell module and test application
+* Multiple stability fixes for the BLE Controller
```

Comment:
Can you add those two from 1.9.1?

Support for PA/LNA amplifiers in the BLE Controller
Support for additional VS commands in the BLE Controller

Reply:
done